src/client/components/ModalWithFeedback/ModalWithFeedback.css

```diff
@@ -0,0 +1,63 @@
+.modal-wrapper {
```

Comment:
You cannot style this here - it will clash with styles defined in src/client/components/Modal/Modal.css.